src/views/layout/default.njk

```diff
@@ -50,14 +51,12 @@ j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
 
 </script> #}
 
-    <div class="header-div-container header-main">
-      <div class="site-width-container">
-        <div class="header-logo-container"><img alt="Department for Environment Food &amp; Rural Affairs Logo" src="{{ assetPath }}/images/defra_logo.svg" width="52" height="42" decoding="async" data-nimg="1" class="defra-logo" loading="lazy" style="color:transparent">
-          <a target="_blank" class="header-logo-link logo-link" href="https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs">
-            <span class="defra-logo-text">Department for Environment <br> Food &amp; Rural Affairs</span></a>
-          <span class="govuk-header__service-name">{{serviceName}}</span></div>
-      </div>
-    </div>
+
+{{ govukHeader({
+  homepageUrl: "https://www.gov.uk/",
+  serviceName: serviceName,
```

Comment:
The service name is not aligned center vertically, it may be due to the font size and line height. The GDS header component aligns the service name vertically centered.

<img width="943" alt="image" src="https://github.com/DEFRA/ncea-frontend/assets/131792244/966d791b-8506-46bb-8b4b-743833ee3cf9">